### ISSUE TITLE
k8s-cloudwatch-adapter to work with multiple regions

### ISSUE THREAD

**Issue by romanudel** [awslabs/k8s-cloudwatch-adapter#30]
Hi I've an EKS cluster that using HPA based on external metrics (the idea to use CRD resources) similar to what described here :
https://aws.amazon.com/blogs/compute/scaling-kubernetes-deployments-with-amazon-cloudwatch-metrics/
The setup i own is that SQS and the  EKS are in two different regions which causing problem getting queries (seems that nothing is returned). What is the correct way to tell hpa where is SQS located( different region)  inorder to get any results for that queue. I don't see any errors but hpa 'targets' remain 0 while the queue keep accumulate messages.

**Comment by chankh** [comment]
Yes, currently the adapter retrieves metrics from the region where it (and the EKS cluster) is running. This is a good feature suggestion, to be able to retrieve metrics from other regions.

**Comment by piotr-struzik** [comment]
Hi 
I'm also impacted by this 
My k8s is running in GKE so no meta-data endpoint available

I tried to pass region as environment variable:
```
- name: AWS_REGION
  value: eu-west-1   
- name: AWS_DEFAULT_REGION
  value: eu-west-1   
``` 
It looks like AWS Go SDK omits this variables and still tries to get the region from "http://169.254.169.254/latest/meta-data/placement/availability-zone/"
This causes adapter to fail to connect and log:
```
I0626 22:19:42.140985       1 client.go:26] using AWS Region: <!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 404 (Not Found)!!1</title>
```

Adding the ability to configure aws region would really help
